NGS2_WITNESS_Cycle1_bayesian_exp1.R

```diff
@@ -0,0 +1,168 @@
+
+#Hypothesis 4.1.4: Rapidly updating strategic networks support cooperation relative to all other conditions 
+#Rand  et al. (2011) coeff = 0.135, p = .006
+
+glm.out.4.1.4.default=stan_glm(decision~fluid_dummy+round_num+
+                   fluid_dummy*round_num, 
+                 data=exp1_cooperation,
+                 family = binomial(link = "logit"), 
+                 chains = 3, iter = 3000)
+bayesian.summary(glm.out.4.1.4.default)
+#launch_shinystan(glm.out.4.1.4)
+
+priors.4.1.4 <- student_t(c(-0.161, -0.171, 0.135), c(0.230, 0.031, 0.050), df=7)
+glm.out.4.1.4.inform=stan_glm(decision~fluid_dummy+round_num+
+                                 fluid_dummy*round_num, 
+                               data=exp1_cooperation,
+                               family = binomial(link = "logit"), 
+                               prior = priors.4.1.4,
+                               prior_intercept = student_t(0.700, 0.122, df = 7),
+                               chains = 3, iter = 3000)
+bayesian.summary(glm.out.4.1.4.inform)
+
+#Hypothesis 4.2.1	Rapidly updating strategic networks have greater network heterogeneity
+
+var <-aggregate(exp1_cooperation[,7], 
+                by=list(sessionnum=exp1_cooperation$sessionnum, 
+                        fluid_dummy=exp1_cooperation$fluid_dummy), 
+                FUN=var)
+fluid.var<-(subset(var$x, var$fluid_dummy==1))
+other.var<-(subset(var$x, var$fluid_dummy==0))
+wilcox.test(fluid.var, other.var, paired = FALSE, alternative = "two.sided")
+### Consider approximate Bayesian alternative, eg. http://andrewgelman.com/2015/07/13/dont-do-the-wilcoxon/
+
+#Hypothesis 4.2.2	Links in rapidly updating strategic networks are more stable between cooperators than between a cooperator and a defector
+#Rand  et al. (2011) coeff = -2.29, p<.000
+
+exp1_rewire.cd.dc<-subset(exp1_rewire, previouslytie==1 & (state=="CC" | state=="CD" | state=="DC"))
+glm.out.4.2.2.default=stan_glm(break.~CC, 
+                       data=exp1_rewire.cd.dc,
+                       family = binomial(link = "logit"), 
+                       chains = 3, iter = 3000)
+bayesian.summary(glm.out.4.2.2.default)
+#launch_shinystan(glm.out.4.2.2.default)
+
+glm.out.4.2.2.inform=stan_glm(break.~CC, 
+                              data=exp1_rewire.cd.dc,
+                              family = binomial(link = "logit"), 
+                              prior = student_t(-2.295, 0.276, df = 7),
+                              prior_intercept = student_t(-0.287, 0.085, df = 7),
+                              chains = 3, iter = 3000)
+bayesian.summary(glm.out.4.2.2.inform)
+#launch_shinystan(glm.out.4.2.2.inform)
+
+#Hypothesis 4.2.3	Links in rapidly updating strategic networks are more stable between cooperators than between between two defectors
+#Rand  et al. (2011) coeff = -2.94, p<.000
+
+exp1_rewire.dd<-subset(exp1_rewire, previouslytie==1 & (state=="CC" | state=="DD"))
+
+glm.out.4.2.3.default=stan_glm(break.~CC, 

```

Comment:
More curious than anything, but what is `break.`? Is that a derived variable of some type or something particular to STAN?